src/docfx/build/toc/BuildTableOfContents.cs

```diff
@@ -4,13 +4,12 @@
 using System.Collections.Generic;
 using System.Diagnostics;
 using System.IO;
-using System.Linq;
 
 namespace Microsoft.Docs.Build
 {
     internal static class BuildTableOfContents
     {
-        public static (IEnumerable<Error>, PublishItem publishItem) Build(Context context, Document file)
+        public static void Build(Context context, Document file)
```

Comment:
>public [](start = 8, length = 6)

How can we avoid to build children during building TOC map? We don't want queue every referenced files when build toc map.

Reply:
Unlike `buildChild`, `WorkQueue.Enqueue` does not trigger build immediately, it just adds to a backing queue, the queue is only build when `Drain` is called.